script.js

```diff
@@ -1,4 +1,90 @@
-var main = function (input) {
-  var myOutputValue = 'hello world';
+var GAME_STATE_DICE_ROLL = 'GAME_STATE_DICE_ROLL';
+var GAME_STATE_CHOOSE_DICE_ORDER = 'GAME_STATE_CHOOSE_DICE_ORDER';
+var GAME_STATE_COMPARE_SCORE = 'GAME_STATE_COMPARE_SCORE';
+var gameState = GAME_STATE_DICE_ROLL;
```

Comment:
Nice usage of what is called a "constant variable" Nice that you put this in SCREAMING_SNAKE_CASE as well. This is the preferred way of doing constants.